app/controllers/city_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CityController < ApplicationController
+  # GET '/cities'
+  def index
+    render json: City.all
```

Comment:
We need to add pagination to this, checkout https://github.com/IcaliaLabs/pager-api for that!

Reply:
I will do it in another branch

Reply:
All right!